test/parallel/test-npm-install.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const common = require('../common');
+
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+
+common.refreshTmpDir();
+
+const npmPath = path.join(
+  common.testDir,
+  '..',
+  'deps',
+  'npm',
+  'bin',
+  'npm-cli.js'
+);
```

Comment:
Ah, I see what you mean.

I'd prefer to not start grabbing package.json and resolving the path from there... lots of extra logic.

If npm changes their entry point this test will break when they send in a PR... it will be a pretty easy fix